src/ga_tx.cpp

```diff
@@ -734,6 +736,14 @@ namespace sdk {
 
                     change = total - required_with_fee;
 
+                    // If a 'greedy' addressee exists send change there
```

Comment:
comment should be inside the if, e.g. `// We have a greedy address: send change there`